github/repos_traffic.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Referrer represent a referrer information
+type Referrer struct {
+	Referrer *string `json:"referrer,omitempty"`
+	Count    *int    `json:"count,omitempty"`
+	Uniques  *int    `json:"uniques,omitempty"`
+}
+
+// Path represent a referrer information
+type Path struct {
+	Path    *string `json:"path,omitempty"`
+	Title   *string `json:"title,omitempty"`
+	Count   *int    `json:"count,omitempty"`
+	Uniques *int    `json:"uniques,omitempty"`
+}
+
+// Time represents a time stamp as in a datapoint
+//
+// This is necessary becausegithub uses unix timestamp and not YYYY-MM-DDTHH:MM:SSZ
+type Time struct {
+	time.Time
+}
```

Comment:
Can [`github.Timestamp`](https://godoc.org/github.com/google/go-github/github#Timestamp) be used, or is this different?